platform.md

```diff
@@ -1088,6 +1099,9 @@ Where:
   "app": [
     {"sha": "<slice-layer-diffID>"}
   ],
+  "bom": {
+    "sha": "<sbom-layer-diffID>"
```

Comment:
We should probably add these keys to the "where" list below.

Reply:
Please see 4f6a803